Modules/TPC/src/Utility.cxx

```diff
@@ -53,6 +55,22 @@ std::vector<TCanvas*> toVector(std::vector<std::unique_ptr<TCanvas>>& input)
   return output;
 }
 
+void fillCanvases(const o2::tpc::CalDet<float>& calDet, std::vector<std::unique_ptr<TCanvas>>& canvases, std::unordered_map<std::string, std::string>& params, const std::string paramName)
+{
+  auto vecPtr = toVector(canvases);
```

Comment:
Perhaps the canvases should be cleared directly before filling?
```suggestion
  auto vecPtr = toVector(canvases);
  clearCanvases(canvases);
```

Reply:
We can do this but `o2::tpc::painter::makeSummaryCanvases` does this anyway in lines 337ff.

Reply:
Very clever ;-) Then we don't need it here, of course.